lib/jekyll/commands/new.rb

```diff
@@ -57,53 +67,29 @@ def initialized_post_name
 
         private
 
-        def gemfile_contents
-          <<-RUBY
-source "https://rubygems.org"
-ruby RUBY_VERSION
-
-# Hello! This is where you manage which Jekyll version is used to run.
-# When you want to use a different version, change it below, save the
-# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
-#
-#     bundle exec jekyll serve
-#
-# This will help ensure the proper Jekyll version is running.
-# Happy Jekylling!
-gem "jekyll", "#{Jekyll::VERSION}"
-
-# This is the default theme for new Jekyll sites. You may change this to anything you like.
-gem "minima"
-
-# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
-# uncomment the line below. To upgrade, run `bundle update github-pages`.
-# gem "github-pages", group: :jekyll_plugins
-
-# If you have any plugins, put them here!
-group :jekyll_plugins do
-   gem "jekyll-feed", "~> 0.6"
-end
-RUBY
-        end
-
         def create_site(new_blog_path)
           create_sample_files new_blog_path
 
           File.open(File.expand_path(initialized_post_name, new_blog_path), "w") do |f|
             f.write(scaffold_post_content)
           end
-
-          File.open(File.expand_path("Gemfile", new_blog_path), "w") do |f|
-            f.write(gemfile_contents)
-          end
         end
 
         def preserve_source_location?(path, options)
           !options["force"] && !Dir["#{path}/**/*"].empty?
         end
 
+        def erb_files
+          erb_file = File.join("*", "*.erb")
+          Dir.glob(erb_file)
+        end
+
         def create_sample_files(path)
           FileUtils.cp_r site_template + "/.", path
+          create_gemfile path
+          erb_files.each do |file|
+            FileUtils.rm file
```

Comment:
> If you don't mind, I'd love to see a `site_template_files` method which returns the value of `Dir.glob` for the `site_template` directory properly excluding `.erb` files and so forth.

@parkr I dont mind at all, infact I learned of the `safe_glob()` method only after seeing the suggested code above. So you're most welcome to point me to the right directions.
The problem with that `site_template_files` method is since it returns all the dotfiles (incl. ".", ".."), the `cp_r` method copies `site_template/.`, `site_template/..` to the `new_blog_path` as well thereby rendering the new method pointless